lib/src/geojson.dart

```diff
@@ -1,33 +1,44 @@
+// ignore_for_file: hash_and_equals
```

Comment:
any context why we need to ignore this, wouldn't it make sense to provide hash/equals for all geojson implementations?

Reply:
Thanks for the hint!
I have implemented the hashCode for the the GeoJSON object types, where I already overloaded the == operator.
I will work on hashCode and == overloading for all types in a different PR